homeassistant/components/verisure/alarm_control_panel.py

```diff
@@ -55,6 +55,16 @@ def unique_id(self) -> str:
         """Return the unique ID for this alarm control panel."""
         return self.coordinator.entry.data[CONF_GIID]
 
+    @property
+    def device_info(self) -> dict[str, Any]:
+        """Return device information about this entity."""
+        return {
+            "name": "Verisure Alarm",
+            "manufacturer": "Verisure",
+            "model": "VBox",
+            "identifiers": {(DOMAIN, self.coordinator.entry.data[CONF_GIID])},
```

Comment:
From the other PRs I saw that we have serial numbers. Is GIID something similar ?

Reply:
GIID is the installation ID, which is like the unique ID for the whole install.
Things like the alarm itself, which is the whole system.